src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -1106,14 +1090,21 @@ def get(self, lv_name=None, vg_name=None, lv_path=None, lv_uuid=None, lv_tags=No
         return lvs[0]
 
 
-def create_lv(name_prefix, uuid, vg=None, device=None, extents=None, size=None, tags=None):
+def create_lv(name_prefix,
+              uuid,
```

Comment:
nit: I find the older way of writing parameters more compact and therefore more readable. We can wrap parameters as follows
```
def somemethod(a, b, c, d, e, f, g, h, i, j, k, l
                            m. n, o):
```

It's legitimate way of writing paraters as per PEP, AFAIR.

Reply:
All true, this style makes for cleaner diffs when arguments are added/removed.